src/components/Button.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const getStyleName = (btn) => {
+  const className = {
+    0: 'equals',
+    '=': 'opt',
+    x: 'opt',
```

Comment:
- [Optional ] Good job for deleting all the unnecessary stuff from the screen but it would be better to delete everything that you are not using like App.test.js,reportWebVitals,setupTest.js ... etc